matrix-sdk/src/main/java/org/matrix/androidsdk/data/DataRetriever.java

```diff
@@ -246,20 +264,21 @@ public void onUnexpectedError(Exception e) {
      * @param token    the start token.
      * @param callback the callback
      */
-    private void forwardPaginate(final IMXStore store, final String roomId, final String token, final ApiCallback<TokensChunkResponse<Event>> callback) {
+    private void forwardPaginate(final IMXStore store, final String roomId,
```

Comment:
Either put all parameters in one line, or put every parameter on its own line (this rule should be applied for functions with 3+ parameters - to be discussed). I do not like to split them like this, I think its harder to read.
Having one parameter per line maybe another solution, especially if we use `final` and nullability annotations.
And Github comparaison/review may be easier because this limited tool only compares and allows comment on full lines.

Reply:
Yes sorry for that, automatic reformat. 
I think we should keep in one line if it fits the limit of 160char.
Otherwise, like you said, one per line.